go.mod

```diff
@@ -109,14 +109,16 @@ require (
 	sigs.k8s.io/yaml v1.1.0
 )
 
-// Holy shit, the docker library versions are a clusterfuck, see https://github.com/moby/moby/issues/39302
+// Docker library versioning is not straightforward, see https://github.com/moby/moby/issues/39302
 // For the moment, the windows build requires a fix that has not been tagged with an official release
 replace github.com/docker/docker => github.com/docker/docker v1.4.2-0.20191213113251-3452f136aa68
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible
 
-replace google.golang.org/grpc => google.golang.org/grpc v1.26.0
+replace google.golang.org/grpc => google.golang.org/grpc v1.29.1
```

Comment:
Is a replace directive necessary here?  We must depend directly on grpc, so should be able to bump the version that way.

Reply:
It seems like the simplest way to pin the dependency given that it's both a direct and indirect dependency, but admittedly I do not know everything about go modules.

Reply:
I can't say for sure without testing, but I believe the version resolution algorithm ensures that only the latest is chosen, so if the main module says 1.29.1, then everything in the program is forced to use that version.  (It changes for v2, though.)

https://golang.org/ref/mod#minimal-version-selection

I would try it and see.  If it doesn't work, submit your change as-is :)